src/app/auth/actions.js

```diff
@@ -18,10 +18,11 @@ export const requestEnable = () => ({
   type: types.REQUEST_ENABLE,
 });
 
-export const receiveEnable = (address, network) => ({
+export const receiveEnable = (address, network, envNetwork) => ({
   type: types.RECEIVE_ENABLE,
   address,
   network,
+  networkMatch: network === envNetwork,
```

Comment:
Actions should not contain logic. They are just constructor functions.

Reply:
Thanks. I moved this logic from the actions file to the operations file with [commit 54eb0d26f below](https://github.com/rnsdomains/rns-manager-react/pull/107/commits/54eb0d26f129c35f8835477c4eeca4b904fdfdb8).